test/lib/test_services.cc

```diff
@@ -148,13 +148,15 @@ future<> table_for_tests::stop() {
 
 namespace sstables {
 
-std::unique_ptr<db::config> make_db_config() {
+std::unique_ptr<db::config> make_db_config(sstring data) {
```

Comment:
I found it confusing that you called this parameter "data". I believe it's the same thing which above you called "dir" or "tempdir". Or, if you want to use the word "data", maybe "data_dir"?

Reply:
Ah, no, I'm OK with either name, just didn't pay much attention to it because it was local parameter for a pretty tiny call. Will rename it shortly.